eclipse-scout-core/src/layout/logicalgrid/LogicalGridLayoutInfo.ts

```diff
@@ -228,77 +230,67 @@ export class LogicalGridLayoutInfo implements LogicalGridLayoutInfoModel {
           prefw = this.logicalWidthInPixel(cons);
         }
         prefw = Math.floor(prefw);
-        for (j = cons.gridx; j < cons.gridx + cons.gridw && j < this.cols; j++) {
+        for (let j = cons.gridx; j < cons.gridx + cons.gridw && j < this.cols; j++) {
```

Comment:
Please simplify this loop. As `cons.gridw` is `1` this loop is pointless and can be replaced by a simple if-statement.
```suggestion
        if (cons.gridx < this.cols) {
```